test/e2e/common/util.go

```diff
@@ -24,3 +28,20 @@ const (
 )
 
 var CurrentSuite Suite
+
+// CommonImageWhiteList is the list of images used in common test. These images should be prepulled
```

Comment:
nit: "These images should be prepulled before a tests starts, so that the tests won't fail due image pulling flakes"